crates/spirv-std/Cargo.toml

```
@@ -12,7 +12,7 @@ bitflags = "1.2.1"
 num-traits = { version = "0.2.14", default-features = false, features = ["libm"] }
 spirv-std-types = { path = "./shared", version = "0.4.0-alpha.15" }
 spirv-std-macros = { path = "./macros", version = "0.4.0-alpha.13" }
-glam = { version = ">=0.17, <=0.21", default-features = false, features = ["libm"], optional = true }
+glam = { version = "=0.22", default-features = false, features = ["libm"], optional = true }
```

```suggestion
glam = { version = "0.22", default-features = false, features = ["libm"], optional = true }
```
(the reason for the operators before was to get a range, effectively)